e2e-tests/seedTimestream.ts

```diff
@@ -0,0 +1,29 @@
+import {
+	WriteRecordsCommand,
+	type _Record,
+} from '@aws-sdk/client-timestream-write'
+import { usageToRecord } from '../lambda/usageToRecord.js'
+import { tsw } from './e2eTest.spec.js'
+
+export const seedTimestream = async (
+	timestamps: Array<Date>,
+	values: Array<number>,
+	iccid: string,
+	dbName: string,
+	tableName: string,
+): Promise<void> => {
+	const records = [] as _Record[]
+	timestamps.forEach((timestamp, index) => {
+		records.push(
+			usageToRecord({ iccid, diff: values[index] ?? 0, currentTime: timestamp })
+				.record,
+		)
+	})
```

Comment:
Please stick to one way of doing loops.

We normally use `for ... in`.

Here you could also do:


```suggestion
	const records = timestamps.map((timestamp, index) => usageToRecord({ iccid, diff: values[index] ?? 0, currentTime: timestamp }).record)
```